packages/e2e-tests/specs/experiments/navigation-editor.test.js

```diff
@@ -347,26 +347,17 @@ describe( 'Navigation editor', () => {
 		);
 		await startEmptyButton.click();
 
-		// NOTE - the following code is commented out.
-		// In CI the editor doesn't seem to support variations.
-		// The following code can be re-introduced once that's resolved.
-		// Add an inner link block.
-		// const appender = await page.waitForSelector(
-		// 	'button[aria-label="Add block"]'
-		// );
-		// await appender.click();
+		const appender = await page.waitForSelector(
+			'button[aria-label="Add block"]'
+		);
+		await appender.click();
 
 		// Must be an exact match to the word 'Link' as other
 		// variations also contain the word 'Link'.
-		// const linkInserterItem = await page.waitForXPath(
-		// 	'//button[@role="option"]//span[.="Link"]'
-		// );
-		// await linkInserterItem.click();
-
-		const appender = await page.waitForSelector(
-			'button[aria-label="Add Link"]'
+		const linkInserterItem = await page.waitForXPath(
+			'//button[@role="option"]//span[.="Link"]'
 		);
-		await appender.click();
+		await linkInserterItem.click();
```

Comment:
Confused as to why the other tests are showing as changed, as these changes are in `trunk` already.

Reply:
I have rebased the trunk now